tools/BUILD.bazel

```diff
@@ -1,8 +1,14 @@
 load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
 
 package(default_visibility = ["//visibility:public"])
 
 ts_config(
     name = "tsconfig",
     src = "//:tsconfig.json",
 )
+
+nodejs_binary(
+    name = "inline-package-json-deps",
+    entry_point = "//tools:inline-package-json-deps.js",
+)
```

Comment:
This can be removed now right?

Reply:
or do you want to keep it for now if we go consider the individual NPM packages in the future?